tests/WCPAY_UnitTestCase.php

```diff
@@ -64,6 +65,13 @@ function( $argument ) use ( $request_class, $request ) {
 			)
 			->willReturn( $response );
 
```

Comment:
I like the idea here, and we can keep it to make it more resilient to changes, but is there a reason why Request Class won't extend parent abstract request class?

Reply:
In many cases, especially for `GET` requests, the consumers just expect an array rather than an object like intentions. Some examples are in my recent PRs: 
- Get_Account https://github.com/Automattic/woocommerce-payments/pull/5879,
- Get_Account_Login_Data and Get_Account_Capital_Link https://github.com/Automattic/woocommerce-payments/pull/5948.